core/src/main/java/greencity/configuration/GreenCityConfigurationProperties.java

```diff
@@ -0,0 +1,32 @@
+package greencity.configuration;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@Getter
+@Setter
+@ConfigurationProperties(prefix = "greencity", ignoreUnknownFields = false)
+public class GreenCityConfigurationProperties {
+    private String tokenKey;
```

Comment:
It's better to have separate property classes for each purpose and do not mis them in single. For example, property class for viberBots, separate for paymentSystems etc